Bonsai.Emergent/CameraIdConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Emergent;
+
+namespace Bonsai.Emergent
+{
+    public class CameraIdConverter : StringConverter
```

Comment:
```suggestion
    public class SerialNumberConverter : StringConverter
```

I would rename this class and file to `SerialNumberConverter` and probably make it internal to avoid exposing internal details to documentation and public API.